packages/models-library/src/models_library/api_schemas_payments/errors.py

```diff
@@ -1,7 +1,7 @@
 from common_library.errors_classes import OsparcErrorMixin
 
 
-class _BaseRpcApiError(OsparcErrorMixin, ValueError):
+class BaseRpcApiError(OsparcErrorMixin, ValueError):
```

Comment:
Q: why the change? This is not used in a different place, else right?

Reply:
yes